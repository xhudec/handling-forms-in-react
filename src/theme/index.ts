export interface ITheme {
  primary: string
}

const theme: ITheme = {
  primary: 'tomato',
}

export default theme
