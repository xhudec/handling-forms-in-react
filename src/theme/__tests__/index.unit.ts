import theme from '..'

describe('theme', () => {
  it('should be defined', () => {
    expect(typeof theme).toBe('object')
  })

  it('should contain theme definitions', () => {
    expect(theme).toMatchSnapshot()
  })
})
