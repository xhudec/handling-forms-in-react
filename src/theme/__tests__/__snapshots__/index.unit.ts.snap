// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme should contain theme definitions 1`] = `
Object {
  "primary": "tomato",
}
`;
